src/main/java/seedu/address/model/rule/Operator.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.rule;
+
+import java.util.Arrays;
+
+/**
+ * Represents an Operator in an ExpressionPredicate.
+ * Guarantees: immutable; is valid as declared in {@link #isValidOperator(String)}
+ */
+public enum Operator {
+    LESS_THAN("LESS THAN");
+
+    private final String representation;
+
+    Operator(String representation) {
+        this.representation = representation;
+    }
+
+    /**
+     * Returns true if a given string is a valid phone number.
```

Comment:
Should the comment be about Operators instead of Phone numbers? haha

Reply:
oh oops. thanks for noticing.